TFT/StandardUnits.txt

```diff
@@ -215,64 +215,64 @@ UPG_MOON_ARMOR	Rema	ELF	upgrade	ML_UPGRADE	HUNTERS_HALL	TREE_AGES	TREE_ETERNITY
 UPG_HIDES	Rerh	ELF	upgrade	ML_UPGRADE	HUNTERS_HALL	TREE_AGES	TREE_ETERNITY	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
 UPG_DRUID_TALON	Redt	ELF	upgrade	ML_UPGRADE	ANCIENT_WIND	TREE_ETERNITY	0	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
 UPG_DRUID_CLAW	Redc	ELF	upgrade	ML_UPGRADE	ANCIENT_LORE	TREE_ETERNITY	0	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-NAGA_SORCERESS	Nngs	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-BEAST_MASTER	Nbst	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-DARK_RANGER	Nbrn	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-NEUTRAL_PIT_LORD	Nplh	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-BREW_MASTER	Npbm	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-GOBLIN_TINKER	Ntin	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-GOBLIN_TINKER_M	Nrob	ELF,HUMAN,ORC,UNDEAD	hero,mutated	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	true	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-FIRELORD	Nfir	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-ALCHEMIST	Nalc	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-ALCHEMIST_M1	Nalm	ELF,HUMAN,ORC,UNDEAD	hero,mutated	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	true	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-ALCHEMIST_M2	Nal2	ELF,HUMAN,ORC,UNDEAD	hero,mutated	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	true	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-ALCHEMIST_M3	Nal3	ELF,HUMAN,ORC,UNDEAD	hero,mutated	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	true	135	2	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+NAGA_SORCERESS	Nngs	NEUTRAL	hero	NEUTRAL_HERO	racial_altar	racial_hall[2]	racial_hall[3]	0	0	false	25	6	0	0	0	0	0	0	0	0	0	0	0	0	0	0

```

Comment:
The available times and restock times should not be divided by 6 in master. Code was changed to use raw values.